src/main/content/try-it.html

```diff
@@ -48,19 +55,22 @@
     <div class="row">
         <div class="col tab-content">
             <!-- Kabanero -->
-            <div class="tab-pane fade show active" id="download_kabanero" role="tabpanel" aria-labelledby="download_kabanero">
+            <!-- <div class="tab-pane fade show" id="download_kabanero" role="tabpanel" aria-labelledby="download_kabanero">
                 <div class="row">
                     <div class="col download_detail">
                         Coming soon!
                     </div>
                 </div>
-            </div>
+            </div> -->
 
             <!-- Codewind -->
-            <div class="tab-pane fade show" id="download_codewind" role="tabpanel" aria-labelledby="download_codewind">
+            <div class="tab-pane fade show active" id="download_codewind" role="tabpanel" aria-labelledby="download_codewind">
                 <div class="row">
                     <div class="col download_detail">
-                        <a href="/codewind">View the Codewind download and install instructions</a>
+                        <p>
+                            Codewind provides the technology that plugs into customers IDEs, that allows container based applications to be build.
```

Comment:
the "that" after the comma doesn't seem to flow correctly and "build" should be "built". Maybe something like either of the 2 examples below

 >Codewind provides technology that plugs into an IDE, which allows container based applications to be built.

 >Codewind provides technology that plugs into an IDE; this allows container based applications to be built.